config.toml

```
@@ -125,6 +125,10 @@ url_latest_version = "https://cdevents.dev/docs"
   version = "latest"
   url = "https://cdevents.dev/docs"
 
+[[params.versions]]
+ version = "v0.1.2"
+ url = "https://github.com/cdevents/spec/tree/v0.1.1/spec.md"
```

I believe this should be 0.1.2, or?
```suggestion
 url = "https://github.com/cdevents/spec/tree/v0.1.2/spec.md"
```

---

Thanks